roscala/src/main/scala/coop/rchain/rosette/utils/package.scala

```diff
@@ -0,0 +1,39 @@
+package coop.rchain.rosette
+
+import java.io.{File, PrintWriter}
+
+import scala.collection.mutable
+
+package object utils {
+
+  def printToFile(f: File)(op: PrintWriter => Unit) {
+    val p = new PrintWriter(f)
+    try { op(p) } finally { p.close() }
+  }
+
+  def pSlice[T](src: mutable.Seq[T], start: Int, end: => Int) =
```

Comment:
Can you briefly describe what you are trying to achieve here?

Reply:
This class is used to provide the C++ pointer semantics for arrays, so each mutation on source affects the result of `pSlice` function.